clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/HostLevelIngestionStats.java

```diff
@@ -87,12 +90,12 @@ public class HostLevelIngestionStats extends AbstractVeniceStats {
    */
   private final Sensor writeComputeCacheHitCount;
 
-  private final LongAdderRateGauge totalLeaderBytesConsumedSensor = new LongAdderRateGauge();
-  private final LongAdderRateGauge totalLeaderRecordsConsumedSensor = new LongAdderRateGauge();
-  private final LongAdderRateGauge totalFollowerBytesConsumedSensor = new LongAdderRateGauge();
-  private final LongAdderRateGauge totalFollowerRecordsConsumedSensor = new LongAdderRateGauge();
-  private final LongAdderRateGauge totalLeaderBytesProducedSensor = new LongAdderRateGauge();
-  private final LongAdderRateGauge totalLeaderRecordsProducedSensor = new LongAdderRateGauge();
+  private final LongAdderRateGauge totalLeaderBytesConsumedSensor;
```

Comment:
can we rename these variables  eg  'byteConsumerRateGauge' or 'byteConsumerRate' to make it clear they are not really sensors .

Reply:
Done. I chose `Rate` as the suffix since some of them were already named this way and it seems fine.